packages/agoric-cli/src/lib/rpc.js

```diff
@@ -153,7 +153,8 @@ export const makeVStorage = (powers, config = networkConfig) => {
             // CosmosSDK ErrUnknownRequest; misrepresentation of the same until
             // https://github.com/Agoric/agoric-sdk/commit/dafc7c1708977aaa55e245dc09a73859cf1df192
             // TODO remove after upgrade-12
-            err.message.match(/unknown request/)
+            err.message.match(/unknown request/) ||
+            err.message.match(/pruned/)
```

Comment:
Ugh, I was really hopeful about **abandoning** error message scans. In this case, the "_failed to load state at height_" text comes from [`func (app *BaseApp) createQueryContext`](https://github.com/agoric-labs/cosmos-sdk/blob/76e151914572fbefb0a6c7ebef1b51d464a9fcbe/baseapp/abci.go#L687-L694), which wraps the "_version mismatch on immutable IAVL tree; version does not exist. Version has either been pruned, or is for a future block height_" text from [`func (st *Store) GetImmutable`](https://github.com/agoric-labs/cosmos-sdk/blob/76e151914572fbefb0a6c7ebef1b51d464a9fcbe/store/iavl/store.go#L114) via [`func (rs *Store) CacheMultiStoreWithVersion`](https://github.com/agoric-labs/cosmos-sdk/blob/76e151914572fbefb0a6c7ebef1b51d464a9fcbe/store/rootmulti/store.go#L497) and unfortunately associates it with an "invalid request" error code. I have opened https://github.com/cosmos/cosmos-sdk/issues/19992 for that issue, and in the meantime I guess we want a corresponding update here.
```suggestion
            // CosmosSDK ErrInvalidRequest with particular message text;
            // misrepresentation of pruned data
            // TODO replace after incorporating a fix to
            // https://github.com/cosmos/cosmos-sdk/issues/19992
            (err.codespace === 'sdk' && err.code === 18 && err.message.match(/pruned/))
```